### ISSUE TITLE
Upgrading process looks a bit weird on AppVeyor

### ISSUE THREAD

**Issue by BergWerkGIS** [felixrieseberg/npm-windows-upgrade#38]
`npm-windows-upgrade` per se works great, but upgrade output on AppVeyor is a bit weird.
Any chance you might consider to look into this rather cosmetic-only request?

https://ci.appveyor.com/project/Mapbox/node-mapnik/build/1.0.839/job/r29p489lwrvbyg68#L25

![image](https://cloud.githubusercontent.com/assets/4549888/10365583/bc7750d0-6dc4-11e5-8f48-7e8c79fb71f2.png)

/cc @FeodorFitsner

**Comment by felixrieseberg** [comment]
Oh, absolutely. I might be too pragmatic, but would it work for you if just disabled the spinner whenever it's run with --no-prompt?

**Comment by BergWerkGIS** [comment]
>  would it work for you if just disabled the spinner whenever it's run with --no-prompt?

Sure, would do.

Maybe I'm missing something, but I already use `npm-windows-upgrade` like this:

```
CALL npm install -g npm-windows-upgrade
CALL npm-windows-upgrade --version:3.3.2 --no-dns-check --no-prompt
```

**Comment by felixrieseberg** [comment]
Sorry, I wasn't clear - *I* would disable the spinner for those who use `--no-prompt`. Patch coming right up :)

**Comment by felixrieseberg** [comment]
...and fixed (in `0.4.5`)! Should work fine now, I'm closing this - let me know if you have any other troubles.

**Comment by BergWerkGIS** [comment]
@felixrieseberg WOW, that was fast, thanks.
Looks a lot cleaner now:
![image](https://cloud.githubusercontent.com/assets/4549888/10387173/732a2920-6e5f-11e5-8891-e8eca914c880.png)
